softwareComponents/kinematics/rofiBot.hpp

```diff
@@ -0,0 +1,109 @@
+#include "calculations.hpp"
+#include <deque>
+#include <cassert>
+#include <math.h>
+
+using chain = std::deque< int >;
+
+constexpr double error = 0.01;
+
+enum class strategy { ccd, fabrik };
+
+class rofi_bot {
```

Comment:
Beware, there is already a class `RoFIBot` in the new configuration on which @jajik is working. Also, this name seems too general for quite a concrete task.